### PR TITLE
[Vectorize] Revisit Learning section

### PR DESCRIPTION
**Pull request by Maddy-Cloudflare**

### CODE REVIEW

File: content/vectorize/learning/what-is-a-vector-database.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ In a traditional vector search use-case, queries are made against a vector datab
...
 2. The output embeddings are inserted into a Vectorize database index.
-3. A search query, classification request or anomaly detection query is also passed through the same ML model, returning an vector embedding representation of the query
-4. Vectorize is queried with this embedding, and returns a set of the most similar vector embeddings to the provided query
-5. The returned embeddings are used to retrieve the original source objects from dedicated storage (e.g. R2, KV, D1) and returned back to the user.
+3. A search query, classification request or anomaly detection query is also passed through the same ML model, returning a vector embedding representation of the query.
+4. Vectorize is queried with this embedding, and returns a set of the most similar vector embeddings to the provided query.
+5. The returned embeddings are used to retrieve the original source objects from dedicated storage (for example, R2, KV, and D1) and returned back to the user.
```
Comment (jason-cf) [inline content/vectorize/learning/what-is-a-vector-database.md:34]:
```suggestion
5. The returned embeddings are used to retrieve the original source objects from dedicated storage (R2, KV, or D1) and returned back to the user.
```

Comment (elithrar) [reply content/vectorize/learning/what-is-a-vector-database.md:34]: This change isn't correct. These are examples. Customers can use any source of truth.

File: content/vectorize/learning/what-is-a-vector-database.md [hunk 2]
```diff
@@ -41,25 +41,25 @@ Retrieval Augmented Generation (RAG) is an approach used to improve the context
...
 
-1. The user prompt is passed into the same ML model used for our dataset, returning a vector embedding representation of the query.
+1. The user prompt is passed into the same ML model used for your dataset, returning a vector embedding representation of the query.
 2. This embedding is used as the query (semantic search) against the vector database, which returns similar vectors.
 3. These vectors are used to look up the content they relate to (if not embedded directly alongside the vectors as metadata).
 4. This content is provided as context alongside the original user prompt, providing additional context to the LLM and allowing it to return an answer that is likely to be far more contextual than the standalone prompt.
 
-Visit the [RAG tutorial using Workers AI](/workers-ai/tutorials/build-a-retrieval-augmented-generation-ai/) to learn how to combine Workers AI and Vectorize for generative AI use-cases.
+Refer to the [RAG tutorial using Workers AI](/workers-ai/tutorials/build-a-retrieval-augmented-generation-ai/) to learn how to combine Workers AI and Vectorize for generative AI use-cases.
```
Comment (jason-cf) [inline content/vectorize/learning/what-is-a-vector-database.md]:
```suggestion
Refer to the [RAG using Workers AI tutorial](/workers-ai/tutorials/build-a-retrieval-augmented-generation-ai/) to learn how to combine Workers AI and Vectorize for generative AI use-cases.
```

File: content/vectorize/learning/what-is-a-vector-database.md [hunk 3]
```diff
@@ -71,23 +71,23 @@ For example, a [text embedding model](/workers-ai/models/text-embeddings/) avail
 [-0.019273685291409492,-0.01913292706012726,<764 dimensions here>,0.0007094172760844231,0.043409910053014755]
 ```
 
-When a model considers the features of an input as "similar" (based on its understanding), the distance between the vector embeddings for those two inputs will typically have a short distance between them.
+When a model considers the features of an input as "similar" (based on its understanding), the distance between the vector embeddings for those two inputs will be short.
 
 ### Dimensions
 
-Vector dimensions describe the width of a vector embedding: the number of floating point elements that comprise a given vector.
+Vector dimensions describe the width of a vector embedding. The width of a vector embegging is the number of floating point elements that comprise a given vector.
```
Comment (jason-cf) [inline content/vectorize/learning/what-is-a-vector-database.md]:
```suggestion
Vector dimensions describe the width of a vector embedding. The width of a vector embedding is the number of floating point elements that comprise a given vector.
```

File: content/vectorize/learning/what-is-a-vector-database.md [hunk 4]
```diff
@@ -71,23 +71,23 @@ For example, a [text embedding model](/workers-ai/models/text-embeddings/) avail
...
 
 The number of dimensions are defined by the machine learning model used to generate the vector embeddings, and how it represents input features based on its internal model and complexity. More dimensions ("wider" vectors) may provide more accuracy at the cost of compute and memory resources, as well as latency (speed) of vector search.
 
 Refer to the [dimensions](/vectorize/learning/create-indexes/#dimensions) documentation to learn how to configure the accepted vector dimension size when creating a Vectorize index.
 
 ### Distance metrics
 
-The distance metric an index uses for vector search defines how it determines how "close" your query vector is to other vectors within the index.
+The distance metric is an index used for vector search. The distance metric defines how it determines how close your query vector is to other vectors within the index.
```
Comment (jason-cf) [inline content/vectorize/learning/what-is-a-vector-database.md]:
```suggestion
The distance metric is an index used for vector search. It defines how it determines how close your query vector is to other vectors within the index.
```

File: content/vectorize/learning/what-is-a-vector-database.md [hunk 5]
```diff
@@ -71,23 +71,23 @@ For example, a [text embedding model](/workers-ai/models/text-embeddings/) avail
...
 
-- Distance metrics determine how the vector search engine assesses “similarity” between vectors.
+- Distance metrics determine how the vector search engine assesses similarity between vectors. 
 - Cosine, Euclidean (L2), and Dot Product are the most commonly used distance metrics in vector search.
-- The machine learning model and type of embedding you use will typically determine which distance metric is best suited for your use-case.
-- Different metrics determine different scoring characteristics. For example, the `cosine` distance metric is well suited to text, sentence similarity and/or document search use-cases; whereas `euclidean` can be better suited to image or speech recognition use-cases.
+- The machine learning model and type of embedding you use will determine which distance metric is best suited for your use-case.
+- Different metrics determine different scoring characteristics. For example, the `cosine` distance metric is well suited to text, sentence similarity and/or document search use-cases. `euclidean` can be better suited for image or speech recognition use-cases.
```
Comment (jason-cf) [inline content/vectorize/learning/what-is-a-vector-database.md:91]:
```suggestion
- Different metrics determine different scoring characteristics. For example, the `cosine` distance metric is well suited to text, sentence similarity and/or document search use-cases. `Euclidean` can be better suited for image or speech recognition use-cases.
```

**Review by jason-cf — APPROVED** [review]

File: content/vectorize/learning/what-is-a-vector-database.md (hunk 5 above)
Comment (elithrar) [reply content/vectorize/learning/what-is-a-vector-database.md:91]: Lowercase `euclidean` is correct here, as it's the setting the user needs to provide.

**Pull request merged**